src/js/modules/core/components/HomePage.js

```diff
@@ -53,7 +54,7 @@ const mapStateToProps = (state) => ({
 });
 
 const mapDispatchToProps = dispatch => {
-    return bindActionCreators({}, dispatch);
+  return bindActionCreators({}, dispatch);
```

Comment:
I was calling the fetch function by using a button but deleted the testing part when i pushed. As for the reason why there is no actions to bind, I think it is just there for future use.